TechAndCheck/zenodotus
# Begin upgrade to Rails 7

This upgrades everything to Rails 7, removing Webpack and updating a bunch of gems to work.

In general this requires two changes from the day to day development:
1. Redis running locally. (see below for setup instructions)
2. Not having to run the dev-server again!

To test this:
Just pull, make sure Redis is running and run the tests.

To install Redis:
MacOS:
1. Using home-brew just `$ brew install redis`
2. `$ redis-server`
3. That's all
